src/test/java/test/abstractmethods/CRUDTest.java

```diff
@@ -2,11 +2,10 @@
 
 import org.testng.annotations.Test;
 
+@Test
 abstract public class CRUDTest {
 
-  @Test
   public abstract void create();
 
-  @Test(dependsOnMethods = "create")
```

Comment:
Fixed this. Realised that this was intentionally introduced by @cbeust as part of [this](https://github.com/cbeust/testng/commit/bb4cfdadd903ebefb0510bc8586b7b353cf29d17) commit. Since my PR is dealing with only `bridged` methods, I reverted my change of excluding `abstract methods`.